test/extended/openstack/bug_1765.go

```diff
@@ -0,0 +1,154 @@
+package openstack
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/gophercloud/gophercloud/v2"
+	"github.com/gophercloud/gophercloud/v2/openstack"
+	"github.com/gophercloud/gophercloud/v2/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/v2/openstack/networking/v2/ports"
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	configv1 "github.com/openshift/api/config/v1"
+	machinev1alpha1 "github.com/openshift/api/machine/v1alpha1"
+	machinev1 "github.com/openshift/api/machine/v1beta1"
+	framework "github.com/openshift/cluster-api-actuator-pkg/pkg/framework"
+	"github.com/openshift/openstack-test/test/extended/openstack/client"
+	"github.com/openshift/openstack-test/test/extended/openstack/machines"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = g.Describe("[sig-installer][Suite:openshift/openstack] Bugfix", func() {
+	defer g.GinkgoRecover()
+
+	var dc dynamic.Interface
+	var clientSet *kubernetes.Clientset
+	var networkClient *gophercloud.ServiceClient
+
+	g.BeforeEach(func(ctx g.SpecContext) {
+		g.By("preparing openshift dynamic client")
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dc, err = dynamic.NewForConfig(cfg)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		clientSet, err = e2e.LoadClientset()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
+
+		g.By("preparing the openstack client")
+		networkClient, err = client.GetServiceClient(ctx, openstack.NewNetworkV2)
+		o.Expect(err).NotTo(o.HaveOccurred(), "Failed to build the OpenStack client")
+	})
+
+	g.Context("bug_1765:", func() {
+		g.It("[Serial] noAllowedAddressPairs on one port should not affect other ports", func(ctx g.SpecContext) {
+			// Check the scenario at https://issues.redhat.com/browse/OCPBUGS-1765
```

Comment:
In other files you'll fine that we are adding the link of the bug that we are automating, like for example: https://github.com/openshift/openstack-test/blob/main/test/extended/openstack/topology.go#L38 

Would you mind to put the comment just before the g.It and with same indentation?

Reply:
Done